day6/spec/day6_spec.rb

```diff
@@ -0,0 +1,28 @@
+require_relative '../day6'
+require_relative '../input'
+
+RSpec.describe do
+  it 'returns correct answer for part 1 example' do
+    day6 = Day6.new(PuzzleExample1)
+    answer = day6.part_one
+    expect(answer).to equal(288)
+  end
+
+  it 'returns correct answer for part 1 input' do
+    day6 = Day6.new(PuzzleInput1)
+    answer = day6.part_one
+    expect(answer).to equal(440000)
```

Comment:
a fun rubyism is underscores are allowed and encouraged in long numbers. You can (and should) write this as `440_000` for readability. Functionally, no difference though, just for the programmer.